skimage/util/_coord_conv.py

```diff
@@ -0,0 +1,234 @@
+import numpy as np
+
+
+def xy_to_rc(points=None, image=None):
+    """Convert points' coordinates and/or image from `x, y` to `r, c` convention.
```

Comment:
Add a sentence to say that the purpose is to flip lines and columns? (If I read this, I understand within a second :) ).